packages/e2e/cypress/e2e/app/formats.cy.ts

```diff
@@ -98,7 +98,7 @@ describe('Explore', () => {
             '£1,999,000',
             '1,999,000 km',
             '1,999,000 mi',
-            '199900000%',
+            '199,900,000%',
```

Comment:
Since now we're using the same formatting function for everything, this included the separators for percent metrics. I believe the goal was to unify everything so the test was changed to match the new format. Do let me know if this should be different 😁